src/mkdocstrings/handlers/python.py

```diff
@@ -204,13 +204,13 @@ def collect(self, identifier: str, config: dict) -> dict:
             raise CollectionError(result["error"])
 
         if result["loading_errors"]:
-            for error in result["loading_errors"]:
-                log.warning(f"mkdocstrings.handlers.python: {error}")
+            for err in result["loading_errors"]:
+                log.warning(f"mkdocstrings.handlers.python: {err}")
```

Comment:
I understand it helps mypy, but I really don't like that. I'd rather have a `# type: ignore` comment.

Reply:
That was actually some other tool complaining (don't remember which) but thats fine